### DISCUSSION TITLE
Feature Suggestion: Visualising Uncertainty

### DISCUSSION THREAD

**Discussion by gimickwu** [royal-statistical-society/datavisguide#53] [Ideas]
Hello,

Thank you for sharing this concise guide to visualisation; I find it gives a nice overview of a field that is the subject of countless books and papers. Do you think that a section on visualising uncertainty could be a crucial addition?

The author of a famous visualisation book once told me "Nobody wants to see uncertainty". I get it, visualising uncertainty often adds challenges and complexity to graphics, in addition to making the message less clear. But could/should uncertainty be in itself part of the message or sometimes the main message in itself? (especially for the Royal Statistical Society) I have the feeling that the main message is sometimes conveyed clearer than it should be, and that may contribute to people thinking that scientists change their minds all the time.

Looking forward to your take on this :)
Cheers

**Comment by nrennie** [comment]
I really like this idea, and visualising uncertainty (without causing unnecessary complexity/confusion/loss of trust) is definitely something that's both challenging and important.
